pages/permission_management.rst

```diff
@@ -0,0 +1,319 @@
+
+Sharing within Teams 
+^^^^^^^^^^^^^^^^^^^^
+
+Before being assigned to a Team, users will see no streams and have no dashboards available. To create a permissions level for a Team, you select the Teams Overview button in the upper righthand corner of the screen. Teams Overview will show you the different Teams you have created in your Graylog environment, including the natural language name and Team description.
+
+.. image:: /images/permissions/permissions_18.png
+
+Graylog uses familiar search mechanisms for sharing views and dashboards so that Admins can more efficiently provide access to resources. 
+
+To add users to a stream, go into the Streams menu. Choose the Stream you want to share. Click “More Actions” and then “Share.”
+
+Once in the Share menu, you can choose to give an individual user or a Team access to the streams. Once you provide access to a Team, all users who are members of that Graylog team will be given access to the Stream. 
+
+When you provide Stream access to a Team, you can also change the permissions for the entire Team. 
+
+As soon as the Admin sets the access for a Team, users in that Team will have the necessary access. 
+
+.. image:: /images/permissions/permissions_19.png
+
+You can choose to add users individually or by their Team. Choosing Security Team provides everyone the same level of access to the Stream all at once rather than adding each user individually:
+
+.. image:: /images/permissions/permissions_20.png
+
+.. image:: /images/permissions/permissions_21.png
+
+Once you save changes, users on the Team automatically gain access to the Stream without needing to log out of Graylog. 
+
+.. image:: /images/permissions/permissions_22.png
+
+Sharing Dashboards within Teams
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Graylog restricts Dashboards to Owners by default, meaning that all newly created Dashboards are “private dashboards.” This default setting ensures that Owners specify who can see their Dashboards and prevents data leakages.  Owners can choose to share Dashboards with individuals or their Teams so that they can collaborate. 
+
+**Example: Bob and Alice**
+
+Alice creates a Dashboard in her account. 
+
+.. image:: /images/permissions/permissions_23.png
+
+Bob, another member of her Team, cannot see the Dashboard in his account because the default Dashboard setting is private.
+
+.. image:: /images/permissions/permissions_24.png
+
+However, Bob can request that Alice share the Dashboard with him so that they can collaborate. When he requests this access, Alice can choose to share only with Bob or with the whole Team. 
+
+Alice then goes to her Dashboard view, chooses the Dashboard she wants to share:
+
+.. image:: /images/permissions/permissions_25.png
+
+Once she chooses the Dashboard, she clicks on the dash in the upper right-hand corner and chooses “Share” from the drop-down menu:
```

Comment:
Changed in 4.0 final:
`Once she chooses the Dashboard, she clicks the "Share" button on the right-hand side in the list:`